### PR TITLE
PR -- Update journal & tutorial-week-4.md

### PR DESCRIPTION
**Pull request by jeremyrist**

### CODE REVIEW

**Comment by boshnivolo** [comment]
Good outline! It's almost poetic :+1:

**Comment by karjac14** [comment]
Last week was really tough and everyone seems become emotional, take a break. let API takeover this week :+1:

**Comment by ericksonk** [comment]
Keep up your positive outlook for this course and you'll do great! Super awesome idea of building a personal website on the side that utilizes what we've learned for further practice :+1:

**Comment by s4dmw** [comment]
journal - good stuff dude. this class/program is overwhelming at times and it helps me to take a step back to get some perspective. i'm constantly reminding myself that it's supposed to be hard as hell and we're supposed to feel like we're drowning most of the time. keep forging ahead!  :+1: 

tutorial - get to poppin them arrays

**Comment by cmauban** [comment]
Hell ya, nice self evaluation :+1: I'm right there with you. This program is really pushing us past our limits so we can be the best we can be. We can do it- keep it up!

File: tutorial-week-4.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 What to do....
 
 ##**ARRAYS!**##
+##POP##
```
Comment (kellymurray) [inline tutorial-week-4.md:4]: Get the draft up!

**Pull request merged**